### PR TITLE
Gradio Status Page 

### PR DESCRIPTION
**Pull request by aliabd**
Created a status page to monitor a few different things: https://status.gradio.app

* Sharing (https://gradio.live) 
* API (https://api.gradio.app/)
* Website (https://gradio.app) 

Will probably change some things around in the page so please give feedback :) 

Also added it to the `COULD_NOT_GET_SHARE_LINK` string message. Which I discovered doesn't actually get printed when there's no internet because we were only catching RuntimeErrors but [this line](https://github.com/gradio-app/gradio/blob/53e4733f2a0cbb9a696a28330dc3927c7997c165/gradio/networking.py#L160) causes a ConnectionError

Closes: #3326

### CODE REVIEW

**Comment by abidlabs** [comment]
Beautiful @aliabd! As commented internally, it would be great if we could somehow combine API & Share links since for end users if either go down, share links go down and it would be a little clearer. Otherwise, LGTM.

**Review by abidlabs — APPROVED** [review]

**Comment by abidlabs** [comment]
Several Spaces went down, causing the flaky tests to fail. Restarting them now

File: gradio/strings.py [hunk 1]
```diff
@@ -10,7 +10,7 @@
     "RUNNING_LOCALLY": "Running on local URL:  {}",
     "RUNNING_LOCALLY_SEPARATED": "Running on local URL:  {}://{}:{}",
     "SHARE_LINK_DISPLAY": "Running on public URL: {}",
-    "COULD_NOT_GET_SHARE_LINK": "\nCould not create share link, please check your internet connection.",
+    "COULD_NOT_GET_SHARE_LINK": "\nCould not create share link. Please check your internet connection or our status page: https://status.gradio.app",
```
Comment (freddyaboulton) [inline gradio/strings.py:13]: I wonder if we should link to this somewhere on the website, too?

**Review by freddyaboulton — APPROVED** [review]
Super cool @aliabd ! Super fast turnaround 🏃‍♂️

File: gradio/strings.py (hunk 1 above)
Comment (abidlabs) [reply gradio/strings.py:13]: Good point, we could add a link in the footer next to the logo: 

image

Comment (abidlabs) [reply gradio/strings.py:13]: image

Comment (aliabd) [reply gradio/strings.py:13]: Added it at the bottom 

Screen Shot 2023-02-27 at 9 24 23 AM

Comment (abidlabs) [reply gradio/strings.py:13]: Hmm not sure but maybe it should be separated a bit more? #notagraphicdesigner but
image

Comment (abidlabs) [reply gradio/strings.py:13]: Or maybe even having it aligned on the right next to twitter / github makes would look nicer color-theme-wise

Comment (abidlabs) [reply gradio/strings.py:13]: LGTM!

**Pull request merged**